docs/content/preview/reference/configuration/yugabyted.md

```diff
@@ -582,6 +582,10 @@ For examples, see [Destroy a local cluster](#destroy-a-local-cluster).
 
 ### finalize_upgrade
 
+{{< warning title="Upgrading to v2.25" >}}
```

Comment:
@nchandrappa this isn't specific to 2.25 though, all upgrades will use the new way right?

Reply:
Yes, that is correct. All new upgrades going forward need to follow these steps.